resources/js/Pages/EventPage.vue

```diff
@@ -13,123 +16,101 @@ import {
   StarIcon,
   CalendarIcon,
   UsersIcon,
-  UserIcon,
 } from '@heroicons/vue/24/outline'
 
-const bannerSrc = 'https://picsum.photos/100/100'
-const eventDate = '26 maja 2025 – 12:00'
-const eventTitle = 'Taylor Swift – City of Lover'
-const venueName = 'Stadion Narodowy'
-const venueAddress = 'al. Księcia Józefa Poniatowskiego 1, 03-901 Warszawa'
-const isPaid = true
+const props = defineProps<{ eventId: number }>()
+
+const event = ref<RawEvent | null>(null)
+const loading = ref(true)
+
+onMounted(async () => {
+  try {
+    const response = await api.get<{ data: RawEvent }>(`/events/${props.eventId}`)
+    event.value = response.data.data
```

Comment:
You can also write like that:

```suggestion
    const { data } = await api.get<{ data: RawEvent }>(`/events/${props.eventId}`)
    
    event.value = data.data
```